docs/literate.jl

```diff
@@ -53,37 +50,33 @@ function preprocess(content)
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
 
-    # adds the current version of the packages
-    append = """
-    # ### Package and system information
-    # #### Package version
+    """ The regex adds "# " at the beginning of each line; chomp removes trailing newlines """
+    literate_format(s) = chomp(replace(s, r"^"m => "# "))
+
+    # <details></details> seems to be buggy in the notebook, so is avoided for now
+    info_footer = """
+    #md # ```@raw html
+    # <details>
+    # <summary><h3>Click to expand package and system information</h3></summary>
```

Comment:
I tried out putting a `<h3>` inside the `<summary>`, but it makes the formatting look weird in the docs.
```suggestion
    # <summary><strong>Package and system information (click to expand)</strong></summary>
```

Reply:
@devmotion have you looked at the docs preview with this change?

Reply:
Yes, I did but it does not address my comments. I think it should be in a separate section, it doesn't belong in the last section of the example and is clearly enough separated visually: https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl/pull/104#issuecomment-1024968468